docs/content/preview/secure/enable-authentication/ysql-create-profiles.md

```diff
@@ -0,0 +1,213 @@
+
+### Lock and unlock roles
+
+To unlock a role that has been locked out, do the following:
+
+```sql
+ALTER ROLE myuser ACCOUNT UNLOCK;
+```
+
+To lock a role so that it can't log in, do the following:
+
+```sql
+ALTER ROLE myuser ACCOUNT LOCK;
+```
+
+### View profiles
+
+The `pg_yb_profile` table lists profiles and their attributes.
+
+To view profiles, enter the following command:
+
+```sql
+SELECT * FROM pg_yb_profile;
+```
+
+You should see output similar to the following:
+
+```output
+  prfname  | prfmaxfailedloginattempts | prfpasswordlocktime 
+-----------+---------------------------+---------------------
+ myprofile |                         3 |                   0
+(1 row)
+```
+
+The following table describes the columns and their values:
+
+| COLUMN | TYPE | DESCRIPTION |
+| :---- | :--- | :---------- |
+| prfname | name | Name of the profile. Must be unique. |
+| prfmaxfailedloginattempts | int | Maximum number of failed attempts allowed. |
+| prfpasswordlocktime | int | Interval in seconds to lock the account. NULL implies that the role will be locked indefinitely. |
+
+### View role profile information
+
+The `pg_yb_role_profile` table lists role profiles and their attributes.
+
+To view role profiles, enter the following command:
+
+```sql
+SELECT * FROM pg_yb_role_profile;
+```
+
+You should see output similar to the following:
+
+```output
+ rolprfrole | rolprfprofile | rolprfstatus | rolprffailedloginattempts | rolprflockeduntil 
+------------+---------------+--------------+---------------------------+-------------------
+      13287 |         16384 | o            |                         0 | 
+(1 row)
+```
+
+The following table describes the columns and their values:
+
+| COLUMN | TYPE | DEFAULT | DESCRIPTION |
+| :----- | :--- | :------ | :---------- |
+| `rolprfrole` | OID | | OID of the row in PG_ROLE
+| `rolprfprofile` | OID | | OID of the row in PROFILE
+| `rolprfstatus` | char | o | The status of the account, as follows:<ul><li>`o` (OPEN); allowed to login.</li><li>`t` (LOCKED(TIMED)); locked for a duration of the timestamp stored in `rolprflockeduntil`. (Note that timed locking is not currently implemented.)</li><li>`l` (LOCKED); locked indefinitely and can only be unlocked by the admin.</li></ul>
+| `rolprffailedloginattempts` | int | 0 | Number of failed attempts by this role.
+| `rolprflockeduntil` | timestamptz | Null | If `rolprfstatus` is `t`, the duration that the role is locked. Otherwise, the value is NULL and not used.
+
+<!-- When login profiles are enabled, you can display these columns in the `pg_roles` table by running the following [meta-command](../../../admin/ysqlsh/#reference):
+
+```sql
+yugabyte=# \dgP
+```
+-->
+
+## Limitations and caveats
+
+- A profile can't be modified using `ALTER PROFILE`. If a profile needs to be modified, create a new profile. ALTER functionality will be implemented in the future.
```

Comment:
If ok to link the GH issue then please link https://github.com/yugabyte/yugabyte-db/issues/15560